common/pom.xml

```diff
@@ -65,6 +65,10 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+        </dependency>
```

Comment:
Why add this dependency? 
If just for test. I think we should change the test case not add dependency.